.git-hooks/commit-msg

```diff
@@ -1,9 +1,9 @@
 #!/bin/sh
 
 test "" = "$(grep '^Signed-off-by: ' "$1" |
-	 sort | uniq -c | sed -e '/^[ 	]*1[ 	]/d')" || {
-	echo >&2 Duplicate Signed-off-by lines.
-	exit 1
+    sort | uniq -c | sed -e '/^[     ]*1[     ]/d')" || {
```

Comment:
This seems a tab vs. space issue.

Do not replace them -- Winery decided to keep tabs to reduce issues at the existing PRs.

Do not add files where you changed only tabs to spaces.